docs/integrations_api.yml

```diff
@@ -374,10 +249,9 @@ paths:
           $ref: "#/responses/InternalServerError"
 
     post:
-      summary: Create firmware image
+      summary: Create a firmware image
```

Comment:
@maciejmrowiec I see what you mean - it's fine for now but I think we will change this once "artifacts" taxonomy is decided on